parts/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -1,9 +1,12 @@
-# Calico Version v2.2.1
-# http://docs.projectcalico.org/v2.2/releases#v2.2.1
+# Calico Version v2.4.1
```

Comment:
Nit - Happy to keep the header comment 5 lines but let's yank all the other comments from yaml in both files.